segmentation_models_pytorch/encoders/efficientnet.py

```diff
@@ -0,0 +1,126 @@
+from efficientnet_pytorch import EfficientNet
+from efficientnet_pytorch.utils import relu_fn, url_map, get_model_params
+import torch.nn as nn
+import torch
+import segmentation_models_pytorch as smp
```

Comment:
Why do we need this import?

Reply:
Just forgot to remove it after porting this code from my "external" implementation :) Fixed.